ts/modules/parallel-coordinates.src.ts

```diff
@@ -315,114 +485,6 @@ extend(ChartProto, /** @lends Highcharts.Chart.prototype */ {
 });
 
 
-// On update, keep parallelPosition.
-Axis.keepProps.push('parallelPosition');
```

Comment:
I think this is necessary?

Reply:
If that is the case, do I have to add the `parallel` property here?

Reply:
Please review.

Reply:
In general, when Axis is updated, it's recreated from scratch. However, some properties are stored (from array `keepProps`) and assigned back.

I don't know if that is still necessary for `parallelPosition` property (or the new one `parallel.position`). Here is the case: https://jsfiddle.net/BlackLabel/c0m8L7d6/3/ - after 1s, updated second yAxis without issues. Here is current build: https://jsfiddle.net/BlackLabel/c0m8L7d6/2/ - btw, first axis is missing.

Reply:
Note that other compositions may not need that property at all.

Reply:
Okay, I fixed it and add compositions to keepProps only, if it makes sense. You fiddle works with the last commit: https://jsfiddle.net/1zu08j4t/